src/main/java/com/hms_networks/americas/sc/extensions/system/time/SCTimeUtils.java

```diff
@@ -147,4 +147,45 @@ public static void injectCustomJvmLocalTime(int timeZoneOffsetMillis) throws Exc
     iso8601UtcTimeFormat = new SimpleDateFormat(utcTimeFormatString, Locale.getDefault());
     iso8601UtcTimeFormat.setTimeZone(TimeZone.getTimeZone(GMT_TIME_ZONE_NAME));
   }
+
+  /**
+   * Returns a string representing the specified time span (in milliseconds) in the format of
+   * '[days] days, [hours] hours, [minutes] minutes, [seconds] seconds'.
```

Comment:
This comment seems to indicate the texts are  fixed but the method takes them as parameters so you are not guaranteed the string "days" or "hours", etc.

Reply:
changed